src/apiCalls.js

```diff
@@ -0,0 +1,14 @@
+const fetchData = (id="") => {
```

Comment:
Ah, maybe this is why you did the parameter assignment? So that you can account for there _not_ being an ID? I think this is fine, but why not just send through the entire url - that way you can control if you're sending an id or not, without this unusual syntax.